anitya/lib/backends/__init__.py

```diff
@@ -265,6 +266,30 @@ def call_url(self, url, insecure=False):
 
             return requests.get(url, headers=headers, verify=not insecure)
 
+    @classmethod
+    def list_recent_projects(self, session):
+        return anitya.lib.model.Project.by_backend(session, self.name)
+
+    @classmethod
+    def update_projects(self):
+        session = anitya.app.SESSION
+        projects = self.list_recent_projects(session)
+        anitya.LOG.info(projects)
+        p = multiprocessing.Pool(anitya.app.APP.config.get('CRON_POOL', 10))
```

Comment:
With a threadpool (instead of a multiproc pool) you might be able to pass the ``session`` object *into* ``update_project`` and thereby avoid re-initializing it for every work item.